about/index.md

```diff
@@ -84,6 +84,19 @@ AGRC is funded through a combination of State general and restricted funds and c
 
 - [Salt Lake User Group (SLUG)](http://www.slug-gis.info/)
 - [Other Local level GIS User Group info]({% link about/organizations/user-groups/index.md %})
+- [National States Geographic Information Council (NGSIC)](https://nsgic.org)
+{: .dotless }
+  </div>
+  <div class="grid__col grid__col--1-of-2" markdown="1">
+###### Utah Geographic Information Systems Advisory Council (GISAC)
+
+- GISAC is composed of individuals from many state agencies.  We are all working together to further the cause of GIS in Utah.  AGRC is trying to create a forum in which all levels of GIS expertise can mingle and learn from each other and learn of each other's challenges. GISAC also sends recommendations to the CIO as potential rules or recommendations that would enhance GIS coordination and cooperation throughout Utah.
+- [Learn more]({% link about/gisac/index.md %})
+{: .dotless }
+  </div>
+  <div class="grid__col grid__col--1-of-2" markdown="1">
+###### National Groups
```

Comment:
**question**: Why doesn't GISAC fit into one of the local, state, or national groups?

Reply:
I see the change from prior now. Can we update this to a 1-of-3 so it tries to fit on one line?

Reply:
There are actually four groups now:
![image](https://user-images.githubusercontent.com/1326248/92008891-f4be8480-ed04-11ea-81ad-b3fbe235139d.png)

Do you want me to change them to 1-of-4?

Reply:
No I don't think we should have 4 groups. Convince me that GISAC should be is its own group when it appears to be a statewide group. Maybe the group titles should be Local, State, and National to fix the organization. 

The section looks off since the others are only lists of links. Since the description is useful, we should add descriptions for the others or remove the descriptions. 

I'm happy to help grab some descriptions from the group websites or somewhere if you also agree.

Reply:
right, since the main title is GIS Organizations & Affiliates the subheadings are then misleading and incorrect in my opinion. Thanks for your work. Since you aren't interested in sorting this out we'll take it from here.

Reply:
I'm happy to do the work, I just didn't know how to reconcile between what you and @brigsz are asking for. I thought that it would be better for you to hash it out directly rather than try and be the middleman.